python/orca/src/bigdl/orca/automl/auto_estimator.py

```diff
@@ -124,21 +132,25 @@ def from_keras(*,
                              name=name)
 
     def fit(self,
-            data,
-            epochs=1,
-            validation_data=None,
-            metric=None,
-            metric_mode=None,
-            metric_threshold=None,
-            n_sampling=1,
-            search_space=None,
-            search_alg=None,
-            search_alg_params=None,
-            scheduler=None,
-            scheduler_params=None,
-            feature_cols=None,
-            label_cols=None,
-            ):
+            data: Union[Callable, Tuple["ndarray", "ndarray"], "DataFrame"],
+            epochs: int=1,
+            validation_data: Optional[
+                Union[Callable, Tuple["ndarray", "ndarray"], "DataFrame"]]=None,
+            metric: Optional[Union[Callable, str]]=None,
+            metric_mode: Optional[str]=None,
+            metric_threshold: Optional[Union["Function", "Float", "Integer"]]=None,
+            n_sampling: int=1,
+            search_space: Optional[Dict[str, Union["Float", "Categorical", "Function", "Integer",
```

Comment:
Just a Optional[Dict] for covering more cases?

Reply:
Ok, It seems that Optional[Dict] is more concise.